dash/_utils.py

```diff
@@ -1,3 +1,11 @@
+def interpolate_str(template, **data):
+    s = template
+    for k, v in data.items():
+        key = '{' + k + '}'
```

Comment:
This is probably too brittle... in modern Javascript you could easily write something like `setConfig({config})` and have someone paste that into `index`

Reply:
Think doubling the brackets `{{config}}` would be enough or maybe add a character before `%{config}` ?

Reply:
doubling is probably not enough... I would do something like `{%dash_config%}` or something personally just to make it super clear